### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by mbdesigns1989**
- I created unit tests for all React components:

- Used Jest snapshots to test the components.

- Used React Testing Library to simulate user interaction.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  expect(operate(one, two, '-')).not.toEqual('70');
+});
+test('the multiplication of one and two to equal 600', () => {
+  expect(operate(one, two, 'x')).toEqual('600');
+});
+test('the multiplication of one and two not to equal 50', () => {
+  expect(operate(one, two, 'x')).not.toEqual('50');
+});
+test('the division of one and two to equal 0.66666666666666666667', () => {
+  expect(operate(one, two, '÷')).toEqual('0.66666666666666666667');
```
Comment (mikenath223) [inline src/__test__/operate.test.js:24]: - [ ] Kindly add a test case for division by zero.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/components/Calculator.js:1]: - [ ] Please remove this eslint disable line.

File: src/components/Match.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/components/Match.js:1]: - [ ] Please remove this eslint disable line.

File: src/components/Math.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/components/Math.js:1]: - [ ] Please remove this eslint disable line.

File: src/components/Nav.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/components/Nav.js:1]: - [ ] Please remove this eslint disable line.

File: src/components/Result.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/components/Result.js:1]: - [ ] Please remove this eslint disable line.

File: src/index.js [hunk 7]
```diff
@@ -1,12 +1,15 @@
-import './index.css';
-
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/index.js:1]: - [ ] Please remove this eslint disable line.

File: src/pages/Home.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/pages/Home.js:1]: - [ ] Please remove this eslint disable line.

File: src/pages/Quote.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
```
Comment (mikenath223) [inline src/pages/Quote.js:1]: - [ ] Please remove this eslint disable line.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @mbdesigns1989 and @Kingjosh007, 

Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
